lib/checkbufferoverrun.cpp

```diff
@@ -383,7 +383,7 @@ static bool checkMinSizes(const std::vector<Library::ArgumentChecks::MinSize> &m
                         parameters.push_back(nullptr);
                 }
                 const MathLib::biguint len = CheckBufferOverrun::countSprintfLength(formatstr, parameters);
-                if (len > arraySize + 2U)
+                if (len > arraySize + 2)
```

Comment:
Simple '2' is better - it means the same in this situation, makes the warning disappear, doesn't raise a question 'why 2U'.